### ISSUE TITLE
DNN Module Action Menus not working after Upgrade to v14.02.x

### ISSUE THREAD

**Issue by jeremy-farrance** [2sic/2sxc#2789]
**I'm submitting a ...**  
[x] bug report 

**...about**  
[x] admin experience UI
[x] DNN parts

**Current behavior**   
If you are in DNN edit mode and use the module action dropdown menu, all the commands do nothing and result in console errors:

![image](https://user-images.githubusercontent.com/653301/174071329-8f4a9e6d-dcdf-4b61-a2d7-2e30eb4a73cb.png)
 

**Expected behavior**   
Clicking 'Edit Template' or 'Admin' should just work.

**Instructions to Reproduce the Problem**
Upgrade a site with DNN 10.09.02 and 2sxc 14.01.x to 2sxc v14.02.00 and the problem is immediate.

**Your environment** 

* **2sxc version(s):** 14.02.00
* **Browser:** all/any
* **DNN:** 9.X.X  (only tested on 10.09.02
* **Language:** any/all

**Anything you would like to add**
Not sure if it is helpful or not, but the link on the 'Edit Template' action menu command is `javascript:$2sxcActionMenuMapper(527).develop();` and that results in this console error:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'sxc')
    at t.run (edit-manager.ts:40:61)
    at n.develop (dnn-inpage-edit.ts:33:28)
    at <anonymous>:1:28
t.run	@	edit-manager.ts:40
n.develop	@	dnn-inpage-edit.ts:33
(anonymous)	@	VM980:1
```

**Comment by jtherrien-accuraty** [comment]
I just did a test by upgrading a fresh install of DNN 9.10.2 from 2sxc 13.12.1 LTS (downloaded and installed both DNN and 2sxc yesterday) to 14.2.0 waiting to install the content app portion until after the update to 14.2.0, and the DNN action menus are throwing those same errors.

**Comment by iJungleboy** [comment]
Thx for finding this. fixed in 14.03.
